projects/igniteui-angular/src/lib/combo/combo.common.ts

```diff
@@ -1202,15 +1202,15 @@ export abstract class IgxComboBaseDirective implements IgxComboBase, AfterViewCh
         this.searchValue = '';
         if (!e.event) {
             this.comboInput?.nativeElement.focus();
+        } else if (this.comboInput?.nativeElement !== this.document.activeElement) {
```

Comment:
PS: Noticed a bit late - assuming this works just fine (seeing it's also validated), but also the check is likely redundant since the combo input shouldn't be focused on close - that's why the check restores it back.

Reply:
Initially, I thought a simple else would be sufficient, but the touched and validation states were still somehow being updated when the if condition was true and the comboInput was focused. Adding this extra check for the active element appears to effectively prevent those updates.

Reply:
Ah, @IvayloG reminded me about https://github.com/IgniteUI/igniteui-angular/pull/14922#discussion_r1818821457
I don't mind the check staying, but I'm still curious in which scenario it's needed, since the open always moves focus to the overlay dropdown, thus on close the input should never be focused and the closing args should have event only on outside click (escape is handled separately).

Reply:
I apologize for the confusion; it turns out that the issue was due to `blur` being triggered while I was in debugging mode. This unintended `blur` was causing the touched and validity states to update unnecessarily.